Tools/CMake/AMReXThirdPartyLibraries.cmake

```diff
@@ -79,9 +79,10 @@ endif ()
 #
 if (AMReX_SUNDIALS)
     if (SUNDIALS_FOUND)
-        message(STATUS "SUNDIALS_FOUND is true, assuming nvecserial or gpu-specific vector found for version 5.7.0")
+        message(STATUS "SUNDIALS_FOUND is true, assuming nvecserial or gpu-specific vector found for version 6.0.0 or higher")
     else ()
-        find_package(SUNDIALS 5.7.0 REQUIRED)
+       set(SUNDIALS_MINIMUM_VERSION 6.0.0 CACHE INTERNAL "Minimum required SUNDIALS version")
+       find_package(SUNDIALS ${SUNDIALS_MINIMUM_VERSION} CONFIG QUIET )
```

Comment:
Same question as above, should we pass components to `find_package` so that `nveccuda` target exists when it is a CUDA build? It is better to raise error here than encounter an error in line 88?

Reply:
It's likely worth checking for components earlier if possible in order to error cleanly. For the case where SUNDIALS might be built with more functionality than we're compiling the amrex application with, we wouldn't want to link anything extra.